src/rules/custom-property-no-empty-line-before/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  isCustomProperty,
+  optionsHaveException,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import _ from "lodash"
+
+export const ruleName = "custom-property-no-empty-line-before"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected single line before custom property",
+  rejected: "Unexpected single line before custom property",
+})
+
+export default function (expectation, options) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        true,
+      ],
```

Comment:
Let's make this Array a single line since there's only one possibility (as is done at line 27 below).